mobile/examples/whisper/android/azure/app/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+    id 'kotlinx-serialization'  // This plug-in is used for parsing json string
+}
+
+android {
+    namespace 'ai.onnxruntime.example.speechrecognition'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "ai.onnxruntime.example.speechrecognition"
```

Comment:
can we use a different applicationId for each example? and possibly the namespace too as it matches applicationId by default.